src/elements/element.rectangle.js

```diff
@@ -50,42 +50,76 @@ module.exports = function(Chart) {
 	}
 
 	Chart.elements.Rectangle = Chart.Element.extend({
-		draw: function() {
+		draw: function(isHorizontalBar) {
```

Comment:
I would keep the rectangle drawing code in `element.rectangle.js` as long as we don't need to use it from somewhere else. For now, I think the solution 1 is the easiest and fastest way to get that PR merged. We can make the rectangle element more generic later. @chartjs/maintainers what do you think?